router/src/config.rs

```diff
@@ -216,6 +216,19 @@ impl Qwen2_5Vl {
     }
 }
 
+#[derive(Clone, Debug, Serialize, Deserialize)]
+#[serde(rename_all = "snake_case")]
+pub struct Gemma3VisionConfig {
+    pub(crate) image_size: usize,
+    pub(crate) patch_size: usize,
+}
+
+#[derive(Clone, Debug, Serialize, Deserialize)]
+#[serde(rename_all = "snake_case")]
```

Comment:
Hmm the name is already `snake_case` do we really need this or is it more like a safeguard?

```suggestion
```